src/settings.kast_blue

```diff
@@ -165,6 +165,7 @@ trace number 0                    # Number of flat frames to use
 trace canbe None                  # If there are frames that will be pixflat ***in addition to other frame types***, include the other frame types here.
 trace lscomb False                # Combine a long and short flat
 trace check condition1 lampstat01=on|lampstat02=on|lampstat03=on|lampstat04=on|lampstat05=on
+trace check condition2 exptime>0  # Required for bias
```

Comment:
You should include the following key at the top of the file:
mosaic camera KASTb